data/data/agent/files/usr/local/bin/install-status.sh

```diff
@@ -1,5 +1,28 @@
 #!/bin/bash
 
+inactive_services() {
+    local services="assisted-service.service create-cluster-and-infraenv.service apply-host-config.service start-cluster-installation.service"
```

Comment:
This seems a little bit fragile. Everytime a developer will add/delete/remove an agent service there's the risk to miss updating this part. It could be useful to have a way to enumerate agent only service, ie by filtering for a fixed unique prefix

Reply:
I somewhat agree, and I thought about templating the list in, but I decided that what we're interested in here is really only a subset of the services - those in the critical chain leading up to start-cluster-installation. So somebody will have to maintain this list somewhere, and it might as well be here. Also, it should be vanishingly rare to add anything to this chain.